k8s/kube-base/job-xdmod-ingestor.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: xdmod-ingestor
+spec:
+  schedule: "* * */1 * *"
```

Comment:
This schedule is going to run the job every minute of every day (because you have `*` for minute (and hour, and month, and day-of-week, and `*/1` means "every one day"). If your intent is to run this once/day, you can write (from [the docs](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-schedule-syntax)):

```
schedule: "@daily"
```

Or if you want a time other than midnight, specify the minutes and hour explicitly. The following will run at 1:00AM every day:

```
schedule: "0 1 * * *"
```

Reply:
When I wrote that, I wanted every 5 minutes, but hadn't settled on that.  however, for testing purposes, every minute was good, though It needs to be once a day or '0 0 * * *'

Reply:
I've updated the cronjob to use the `@daily` schedule.